projects/1.18/assets/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+{
+  "block.butchercraft.bbq_ribs_block": "炭烤排骨",
+  "block.butchercraft.beef_block": "生牛肉块",
+  "block.butchercraft.beef_wellington_block": "惠灵顿牛排",
+  "block.butchercraft.butcher_block": "案板",
+  "block.butchercraft.cooked_beef_block": "熟牛肉块",
+  "block.butchercraft.cooked_lamb_block": "熟羊肉块",
+  "block.butchercraft.cooked_pork_block": "熟猪肉块",
+  "block.butchercraft.cow_hide_carpet": "牛皮",
+  "block.butchercraft.drying_rack": "晾干架",
+  "block.butchercraft.haggis_block": "哈吉斯",
+  "block.butchercraft.hash_block": "肉丁土豆",
+  "block.butchercraft.jelly_black_block": "黑莓果冻",
+  "block.butchercraft.jelly_blue_block": "蓝莓果冻",
+  "block.butchercraft.jelly_brown_block": "可乐果冻",
+  "block.butchercraft.jelly_cyan_block": "潘趣果冻",
+  "block.butchercraft.jelly_green_block": "苹果果冻",
+  "block.butchercraft.jelly_grey_block": "神秘果冻",
+  "block.butchercraft.jelly_light_blue_block": "蓝色覆盆子果冻",
+  "block.butchercraft.jelly_light_grey_block": "火龙果果冻",
+  "block.butchercraft.jelly_lime_block": "酸橙果冻",
+  "block.butchercraft.jelly_magenta_block": "杂莓果冻",
+  "block.butchercraft.jelly_orange_block": "橘子果冻",
+  "block.butchercraft.jelly_pink_block": "草莓果冻",
+  "block.butchercraft.jelly_purple_block": "葡萄果冻",
+  "block.butchercraft.jelly_red_block": "樱桃果冻",
+  "block.butchercraft.jelly_white_block": "椰子果冻",
+  "block.butchercraft.jelly_yellow_block": "柠檬果冻",
+  "block.butchercraft.lamb_block": "生羊肉块",
+  "block.butchercraft.mashed_potato_gravy_block": "肉汁土豆泥",
+  "block.butchercraft.meat_hook": "挂肉钩",
+  "block.butchercraft.meat_pie": "肉饼",
+  "block.butchercraft.pig_hide_carpet": "猪皮",
+  "block.butchercraft.pork_block": "生猪肉块",
+  "block.butchercraft.potroast_block": "罐焖牛肉",
+  "block.butchercraft.pulled_pork_block": "手撕猪肉",
+  "block.butchercraft.rack_lamb_block": "烤羊排",
+  "block.butchercraft.salisbury_steak_block": "索尔兹伯里牛排",
+  "block.butchercraft.sheep_hide_carpet": "羊皮",
+  "block.butchercraft.stirfry_block": "蔬菜炒肉丝",
+  "butchercraft.advancement.bonesaw.desc": "制作一把骨锯",
+  "butchercraft.advancement.bonesaw.name": "骷髅的噩梦。",
+  "butchercraft.advancement.butcherknife.desc": "制作一把屠刀",
+  "butchercraft.advancement.butcherknife.name": "屠夫不杀五爪猪",
+  "butchercraft.advancement.cannibalism.desc": "试试去宰了一个人类。",
+  "butchercraft.advancement.cannibalism.name": "这里不是《环世界》……",
```

```suggestion
  "butchercraft.advancement.cannibalism.name": "这里不是《边缘》……",
```
官方译名为`边缘世界`，`环世界`已确定为错译

---

我个人觉得这个不用改。现在依旧有很多人叫这个旧译名。

---

就像《Elden Ring》还有人称呼为“老头环”一样，我觉得这种玩家约定俗成的译名反而更适合这里的语境；
如果要改，那就
```suggestion
  "butchercraft.advancement.cannibalism.name": "这里不是《边缘世界》……",
```